### ISSUE TITLE
jog config: can't dup NilClass

### ISSUE THREAD

**Issue by cdlm** [tysonmote/jog#1]
After a fresh install, I'm getting the error below, because `DEFAULT_TEMPLATE` is nil. Changing it to an empty string seems to fix the problem, but I'm wondering it setting it to the template shown in the documentation would make more sense.

```
jog-0.0.5/lib/jog/config.rb:49:in `dup': can't dup NilClass (TypeError)
```

**Comment by tysonmote** [comment]
That sounds like a sensible solution -- I'll push a fix later today or tomorrow. Thanks for the heads up!

**Comment by tysonmote** [comment]
Fixed in 0.0.6. Thanks!
